<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LogFileViewerFilterEditorDockWidget</class>
 <widget class="QDockWidget" name="LogFileViewerFilterEditorDockWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>425</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DockWidget</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QWidget" name="control" native="true">
      <property name="locale">
       <locale language="English" country="UnitedStates"/>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string>Color</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QGroupBox" name="groupBox">
         <property name="locale">
          <locale language="English" country="UnitedStates"/>
         </property>
         <property name="title">
          <string>Date</string>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <widget class="QLabel" name="labelDateFrom">
            <property name="locale">
             <locale language="English" country="UnitedStates"/>
            </property>
            <property name="text">
             <string>From</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QDateTimeEdit" name="dateTimeEditFrom">
            <property name="locale">
             <locale language="English" country="UnitedStates"/>
            </property>
            <property name="date">
             <date>
              <year>2009</year>
              <month>1</month>
              <day>1</day>
             </date>
            </property>
            <property name="displayFormat">
             <string>dd.MM.yyyy HH:mm:ss:zzz</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="labelDateTo">
            <property name="locale">
             <locale language="English" country="UnitedStates"/>
            </property>
            <property name="text">
             <string>to</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QDateTimeEdit" name="dateTimeEditTo">
            <property name="locale">
             <locale language="English" country="UnitedStates"/>
            </property>
            <property name="date">
             <date>
              <year>2010</year>
              <month>1</month>
              <day>1</day>
             </date>
            </property>
            <property name="displayFormat">
             <string>dd.MM.yyyy HH:mm:ss:zzz</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QGroupBox" name="groupBoxFilter">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="locale">
          <locale language="English" country="UnitedStates"/>
         </property>
         <property name="title">
          <string>Filter</string>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="0" column="0">
           <widget class="QLabel" name="labelSeverity">
            <property name="locale">
             <locale language="English" country="UnitedStates"/>
            </property>
            <property name="text">
             <string>Severity</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="labelFacility">
            <property name="locale">
             <locale language="English" country="UnitedStates"/>
            </property>
            <property name="text">
             <string>Facility</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="labelSourceID">
            <property name="locale">
             <locale language="English" country="UnitedStates"/>
            </property>
            <property name="text">
             <string>SourceID</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="labelMessageID">
            <property name="locale">
             <locale language="English" country="UnitedStates"/>
            </property>
            <property name="text">
             <string>MessageID</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QComboBox" name="comboBoxSeverity">
            <item>
             <property name="text">
              <string>All</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Action</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Information</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Telegram</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Warning</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Recoverable</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Error</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Fatal</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QComboBox" name="comboBoxFacility">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <item>
             <property name="text">
              <string>Developer</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Service</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Customer</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="lineEditSourceID"/>
          </item>
          <item row="3" column="1">
           <widget class="QLineEdit" name="lineEditMessageID"/>
          </item>
         </layout>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QGroupBox" name="groupBoxSearch">
         <property name="title">
          <string>Search</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_6">
          <item row="0" column="0">
           <widget class="QLineEdit" name="lineEditSearch"/>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>LogFileViewerFilterEditorDockWidget</receiver>
   <slot>setColor()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>314</x>
     <y>53</y>
    </hint>
    <hint type="destinationlabel">
     <x>397</x>
     <y>61</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dateTimeEditFrom</sender>
   <signal>dateTimeChanged(QDateTime)</signal>
   <receiver>LogFileViewerFilterEditorDockWidget</receiver>
   <slot>setFrom(QDateTime)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>355</x>
     <y>121</y>
    </hint>
    <hint type="destinationlabel">
     <x>395</x>
     <y>126</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dateTimeEditTo</sender>
   <signal>dateTimeChanged(QDateTime)</signal>
   <receiver>LogFileViewerFilterEditorDockWidget</receiver>
   <slot>setTo(QDateTime)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>344</x>
     <y>167</y>
    </hint>
    <hint type="destinationlabel">
     <x>398</x>
     <y>171</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>setColor()</slot>
  <slot>setFrom(QDateTime)</slot>
  <slot>setTo(QDateTime)</slot>
 </slots>
</ui>
